crates/router/src/connector/paybox/transformers.rs

```diff
@@ -83,8 +95,37 @@ pub struct PayboxPaymentsRequest {
 
     #[serde(rename = "CLE")]
     pub key: Secret<String>,
-}
 
+    #[serde(rename = "ID3D")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub three_ds_data: Option<Secret<String>>,
+}
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct ThreeDSPaymentsRequest {
+    id_merchant: Secret<String>,
+    id_session: String,
+    amount: MinorUnit,
+    currency: String,
+    #[serde(rename = "CCNumber")]
+    cc_number: cards::CardNumber,
+    #[serde(rename = "CCExpDate")]
+    cc_exp_date: Secret<String>,
+    #[serde(rename = "CVVCode")]
+    cvv_code: Secret<String>,
+    #[serde(rename = "URLRetour")]
+    url_retour: String,
+    #[serde(rename = "URLHttpDirect")]
+    url_http_direct: String,
+    email_porteur: common_utils::pii::Email,
+    first_name: Secret<String>,
+    last_name: Secret<String>,
+    address1: Secret<String>,
+    zip_code: Secret<String>,
+    city: String,
+    country_code: api_models::enums::CountryAlpha2,
+    total_quantity: i32,
+}
```

Comment:
Make only required fields as mandatory and add the in `crates/router/src/configs/defaults.rs`.
Make strongly recommended fields as optional and add a comment highlighting those fields.